src/DashboardUI/src/data-contracts/CombinedPointData.ts

```diff
@@ -0,0 +1,12 @@
+import { GeometryEtDatapoint } from './GeometryEtDatapoint';
+
+type FullPointData = {
```

Comment:
We should probably match the backend data contract naming if we can. Does this name make sense in this context?

```suggestion
interface PointEtDataCollection {
```

Prolly stick to `interface` unless we're using the `type` specific features (extension / unions / etc)

Reply:
1. we already have the `PointEtDataCollection` contract in the frontend

2. this one's a bit weird. I'm using the types in this file to reference control location data in state. I'm doing this because the data in state can come from three sources, which have a mix of properties:

* call chain: estimate consumptive use
    * returns `PointEtDataCollection`
* call chain: get application
    * returns `ControlLocationDetails`
* interaction: user selects map on polygon
    * returns `MapSelectionPointData` (defined in this file)

This also follows the convention (if you can call it that) of the `CombinedPolygonData.ts` file, which implements the same types for polygons to be tracked in state.